lib/ansible/modules/system/java_keystore.py

```diff
@@ -200,8 +205,8 @@ def create_jks(module, name, openssl_bin, keytool_bin, keystore_path, password):
             if os.path.exists(keystore_p12_path):
                 os.remove(keystore_p12_path)
 
-            export_p12_cmd = "%s pkcs12 -export -name '%s' -in '%s' -inkey '%s' -out '%s' -passout 'pass:%s'" % (
-                openssl_bin, name, certificate_path, private_key_path, keystore_p12_path, password)
+            export_p12_cmd = "%s pkcs12 -export -name '%s' -in '%s' -inkey '%s' -out '%s' -passout 'pass:%s' -passin 'pass:%s'" % (
```

Comment:
Are you sure it's still working when there's no passphrase ?
In other words I don't think we should add the command line option `-passin" if there's no passphrase.